honeybee_schema/radiance/asset.py

```diff
@@ -77,6 +77,14 @@ class SensorGrid(_RadianceAsset):
         'of sensors or the mesh is too large to be practically visualized.'
     )
 
+    group_identifier: str = Field(
```

Comment:
Why not just add it once to the base class of `_RadianceAsset`?

Reply:
Hey @chriswmackey, only because of different description. Should I just remove grids and views from description?